scipy/sparse/linalg/_eigen/tests/test_svds.py

```diff
@@ -316,17 +304,15 @@ def err(tol):
         # for 'arpack' and 'propack', accuracies make discrete steps
         accuracies = {'propack': [1e-12, 1e-6, 1e-4],
                       'arpack': [2e-15, 1e-10, 1e-10],
-                      'lobpcg': [1e-11, 1e-3, 10]}
+                      'lobpcg': [1e-10, 1e-3, 10]}
 
         for tol, accuracy in zip(tols, accuracies[self.solver]):
             error = err(tol)
             assert error < accuracy
-            assert error > accuracy/10
+            # Why do we test a lower bound for error?
```

Comment:
Agreed. that doesn't many too much sense. I'll remove it now that I'm touching this test anyway.